### PR TITLE
Proposing expand description

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/office-docs-powershell/issues/7742

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md [hunk 1]
```diff
@@ -827,9 +827,13 @@ Accept wildcard characters: False
 
 ### -ConditionalAccessPolicy
 
-Please read [Control access from unmanaged devices](https://docs.microsoft.com/sharepoint/control-access-from-unmanaged-devices ) documentation here to understand Conditional Access Policy usage in SharePoint Online.
+This parameter either allows full access from desktop apps, mobile apps, and the web (AllowFullAccess), Allows limited, web-only access (AllowLimitedAccess) or Blocks Access (BlockAccess). Please read [Control access from unmanaged devices](https://docs.microsoft.com/sharepoint/control-access-from-unmanaged-devices ) documentation here to understand Conditional Access Policy usage in SharePoint Online.
```
Comment (dariomws) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md:830]:
```suggestion
This parameter either allows full access from desktop apps, mobile apps, and the web (AllowFullAccess), Allows limited, web-only access (AllowLimitedAccess) or Blocks Access (BlockAccess). Please read [Control access from unmanaged devices](/sharepoint/control-access-from-unmanaged-devices) documentation here to understand Conditional Access Policy usage in SharePoint Online.
```

**Review by dariomws — APPROVED** [review]
I think we should use site-relative links.

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md (hunk 1 above)
Comment (tseward) [reply sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md:830]: Breaking this up into multiple sentences would be better, IMO. "AllowedFullAccess allows full access from desktop [...]. AllowedLimitedAccess allows web-only access [...]..."

**Review by tseward — CHANGES_REQUESTED** [review]

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Kateyanne Please merge this PR. Thanks

**Pull request merged**